src/components/problem/edit.vue

```diff
@@ -1,44 +1,84 @@
+          <el-checkbox v-model="disable" label="Disabled" class="item"></el-checkbox>
+          <el-checkbox v-model="html" label="Allow HTML" class="item"></el-checkbox>
+        </el-card>
+        <el-card class="item">
+          <i class="el-icon-menu" /> Limitation
+          <el-divider>Time</el-divider>
+          <el-row :gutter="10">
+            <el-col :span="20">
+              <el-input v-model="time" placeholder="Set time limitation"></el-input>
+            </el-col>
+            <el-col :span="4" class="center-text">
+              MS
+            </el-col>
+          </el-row>
+          <el-divider>Memery</el-divider>
+          <el-row :gutter="10">
+            <el-col :span="20">
+              <el-input v-model="memery" placeholder="Set memery limitation"></el-input>
+            </el-col>
+            <el-col :span="4" class="center-text">
+              MB
+            </el-col>
+          </el-row>
+        </el-card>
+        <el-card class="item">
+          <el-button type="primary" @click="submit();" :loading="buttonLoading">Submit</el-button>
+          <el-button @click="back();">Back</el-button>
+        </el-card>
+      </el-col>
+      <el-col :span="18">
+        <el-card v-loading="contentLoading">
+          <div slot="header" class="clearfix"><i class="el-icon-document" /> Content</div>
+          <MarkdownEditor v-model="mdContent" />
+        </el-card>
+      </el-col>
+    </el-row>
+  </div>
+  <div v-else>
+    <el-card class="float">
+      <el-button type="primary" @click="submit();" icon="el-icon-check" circle />
+      <el-button @click="back();" icon="el-icon-back" circle />
+    </el-card>
+    <el-card>
+      <div slot="header" class="clearfix"><i class="el-icon-edit-outline" /> Name</div>
+      <el-input v-model="title" placeholder="Input problem title here"></el-input>
+      <el-checkbox v-model="disable" label="Disabled" class="item"></el-checkbox>
+      <el-checkbox v-model="html" label="Allow HTML" class="item"></el-checkbox>
+    </el-card>
+    <el-card class="item">
+      <i class="el-icon-menu" /> Limitation
+      <el-divider>Time</el-divider>
+      <el-row :gutter="10">
+        <el-col :span="20">
+          <el-input v-model="time" placeholder="Set time limitation"></el-input>
+        </el-col>
+        <el-col :span="4" class="center-text">
+          MS
+        </el-col>
+      </el-row>
+      <el-divider>Memery</el-divider>
+      <el-row :gutter="10">
+        <el-col :span="20">
+          <el-input v-model="memery" placeholder="Set memery limitation"></el-input>
+        </el-col>
+        <el-col :span="4" class="center-text">
+          MB
+        </el-col>
+      </el-row>
+    </el-card>
+    <el-card v-loading="contentLoading" class="item">
+      <div slot="header" class="clearfix"><i class="el-icon-document" /> Content</div>
+      <MarkdownEditor v-model="mdContent" />
+    </el-card>
+  </div>
```

Comment:
This file is too complex too, **please add some comment to it**.
Also, you can refactor this file by split different logic into different files, For example:

- `edit_mobile.vue`
- `edit_desktop.vue`

or even:

- `EditTags.vue`
- `EditContent.vue`
- `EditNameAndPID.vue`